xbmc/cores/VideoPlayer/VideoRenderers/BaseRenderer.h

```diff
@@ -85,6 +85,7 @@ class CBaseRenderer
   */
   void GetVideoRect(CRect& source, CRect& dest, CRect& view) const;
   float GetAspectRatio() const;
+  unsigned int GetOrientation() { return m_renderOrientation; }
```

Comment:
Could this method be `const`?

Reply:
sure. done, along with the format fixes.